packages/medusa-payment-stripe/src/api/utils/utils.ts

```diff
@@ -251,8 +251,8 @@ async function completeCartIfNecessary({
     if (response_code !== 200) {
       throw new MedusaError(
         MedusaError.Types.UNEXPECTED_STATE,
-        response_body["message"],
-        response_body["code"].toString()
+        response_body["message"] as string,
```

Comment:
weird, I can understand since reponse_body is of type object. But why the build didn't shown it earlier that is the question. maybe because now there is the version 4.9.5 of ts in the dev deps.